Geo-Gecko/DES_dashboard_project
# Download functionality

#### What does this PR do?
Adds download buttons to the different cards on each page

#### How should this be manually tested?
Run `git fetch` and `git checkout download-functionality` . Run  `npm start` to see the functionality in action.

#### Any background context you want to provide?
The above commands should be run from a bash terminal such that one can easily export database environment variables from a .env file
